plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator.go

```diff
@@ -25,15 +25,14 @@ import (
 
 // HelmChartInflationGeneratorPlugin is a plugin to generate resources
```

Comment:
Can we omit this reference to `HelmChartInflationGeneratorPlugin` in the comment since we changed the `struct` `type` name? 

Seems like the other transformer comments start with a verb.

Reply:
done!